samples/gradle-plugin/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+tasks {
+
+    val publishPlugin by creating(GradleBuild::class) {
+        dir = file("plugin")
+        tasks = listOf("publish")
+    }
+
+    val checkConsumer by creating(GradleBuild::class) {
```

Comment:
I'm a bit bothered by the `check` prefix here.

How about renaming `publishPlugin` to just `plugin` and `checkConsumer` to just `consumer`?